src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
+@mixin center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+@keyframes rotate-block {
+  from {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
+
 body {
   margin: 0;
 }
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  @include center;
+
+  &__minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+
+    transform-origin: 50% 100%;
+    animation: rotate-block 3600s steps(60, end) infinite;
+
+    @include center;
+  }
+
+  &__seconds {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+
+    transform-origin: 50% 100%;
+    animation: rotate-block 60s linear infinite normal;
```

Comment:
The `animation` shorthand for `.stopwatch__seconds` is correct for a stopwatch second hand. However, the keyword `normal` is not necessary as it is the default value for the `animation-direction` property and can be omitted.